sabnzbd/skintext.py

```diff
@@ -495,9 +495,13 @@
     "nodupes-tag": TT("Tag job"),  #: Four way switch for duplicates
     "abort": TT("Abort"),  #: Three way switch for encrypted posts
     "opt-action_on_unwanted_extensions": TT("Action when unwanted extension detected"),
-    "explain-action_on_unwanted_extensions": TT("Action when an unwanted extension is detected in RAR files"),
+    "explain-action_on_unwanted_extensions": TT("Action when an unwanted extension is detected"),
```

Comment:
Do we need to change the translation? Because it is detected in RAR files, either during downloading or after extraction. So the text isn't wrong.

Reply:
It applies to filenames that aren't rar or in rar files too, even before this change. See [nzbstuff.py:891](https://github.com/sabnzbd/sabnzbd/blob/10c56e08d4a7333a26f4a435025c314b4ce343f9/sabnzbd/nzbstuff.py#L891)